kalite/distributed/static/js/distributed/exercises/api_resources.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.http import HttpResponse
+
+from tastypie import fields
+from tastypie.resources import ModelResource
+
+import yaml
```

Comment:
After a bunch of research, looks like `PyYAML` can run as a pure python module, given that it's not compiled with `libyaml` bindings. We can force that with `pip install pyyaml --global-option="--without-libyaml"`.